package.json

```diff
@@ -18,30 +18,33 @@
   ],
   "main": "src/index.js",
   "scripts": {
+    "format": "npm run format:js && npm run format:md",
+    "format:js": "prettier --write '**/*.js'",
+    "format:md": "prettier --write --parser markdown '**/*.md'",
```

Comment:
These lines seem unrelated to the goal of the pull request.

Reply:
Yeah I can remove them if you want.

Reply:
Yes please, they seem unrelated.